### PR TITLE
Making Raw and Release notes data more intuitive for package and packages data source

### PR DESCRIPTION
**Pull request by Shubbhang351**
1. **What this PR does / why we need it**:

 Making Raw and Release notes data more intuitive for package and packages data source
 
 Result :- 
 
 
Screenshot 2023-10-17 at 2 44 32 PM

--------------------------------------------------

Acceptance tests

Screenshot 2023-10-18 at 11 20 37 AM

----------------------------------------------------

Screenshot 2023-10-18 at 11 20 43 AM

### CODE REVIEW

File: internal/resources/package/spec/cluster_scope.go [hunk 1]
```diff
@@ -24,9 +26,16 @@ func FlattenSpecForClusterScope(spec *packageclustermodel.VmwareTanzuManageV1alp
 
 	var valuedata = make(map[string]interface{})
 
+	var releaseNotes = make(map[string]interface{})
+
 	flattenSpecData[ReleasedAtKey] = (spec.ReleasedAt).String()
 	flattenSpecData[CapacityRequirementsDescriptionKey] = spec.CapacityRequirementsDescription
-	flattenSpecData[ReleaseNotesKey] = spec.ReleaseNotes
+	releaseNotesValue := strings.Split(spec.ReleaseNotes, " ")
```
Comment (ishangupta-ds) [inline internal/resources/package/spec/cluster_scope.go:33]: Use constant

File: internal/resources/package/spec/spec.go [hunk 2]
```diff
@@ -64,10 +60,85 @@ var (
...
+											Computed: true,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				titleKey: {
+					Type:     schema.TypeString,
```
Comment (ishangupta-ds) [inline internal/resources/package/spec/spec.go:140]: Description?

File: internal/resources/package/spec/spec_flatten_test.go [hunk 3]
```diff
@@ -50,13 +50,39 @@ func TestFlattenSpecForClusterScope(t *testing.T) {
...
+												map[string]interface{}{
+													namespaceKey: []interface{}{
+														map[string]interface{}{
+															defaultKey:     "cert-manager",
+															descriptionKey: "The namespace in which to deploy cert-manager.",
+															typeKey:        "string",
+														},
+													},
+												},
+											},
```
Comment (ishangupta-ds) [inline internal/resources/package/spec/spec_flatten_test.go:82]: Check indentation.

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: internal/resources/package/spec/cluster_scope.go (hunk 1 above)
Comment (Shubbhang351) [reply internal/resources/package/spec/cluster_scope.go]: Updated PTAL

File: internal/resources/package/spec/spec.go (hunk 2 above)
Comment (Shubbhang351) [reply internal/resources/package/spec/spec.go]: Updated PTAL

File: internal/resources/package/spec/spec_flatten_test.go (hunk 3 above)
Comment (Shubbhang351) [reply internal/resources/package/spec/spec_flatten_test.go:82]: I've checked it and the indentation seems to be fine. PTAL

**Comment by ishangupta-ds** [comment]
Don't the acceptance tests need any update?

**Review by shobha2626 — APPROVED** [review]
LGTM

**Review by ishangupta-ds — APPROVED** [review]

**Review by ramya-bangera — APPROVED** [review]

**Pull request merged**